### ISSUE TITLE
Why amplitude only?

### ISSUE THREAD

**Issue by ayyp1** [ludlows/CSI-Activity-Recognition#4]
Hi it might be a silly question but why did you take the amplitude only for model if the data has both amplitude and phase?
or am I missing something?

**Comment by FirefoxSRV** [comment]
No, the phase only accounts for the different bounces of the radio waves....u can't classify with phase if my perception is correct
